Warn if a suggestion does not have the parent attribute

quodlibetor opened the pull request:
A customer has reported this happening and some of us were able to reproduce
it, but no one can reproduce it currently.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/MaterializeInc/mzcli?pullRequest=34) <br/>All committers have signed the CLA.

quodlibetor committed:
Warn if a suggestion does not have the parent attribute

A customer has reported this happening and some of us were able to reproduce
it, we but no longer are.

quodlibetor committed:
fix test skew from upstream
